MockObjects/Objects/MockObjectsFiles/MockRepositories.hpp

```diff
@@ -0,0 +1,108 @@
+#pragma once 
+
+#include <DataAccess/IAdapter.hpp>
+#include <DataAccess/IServerRepositories.hpp>
+
+#include "MockQuery.hpp"
+
+namespace MockRepositories
+{
+using IAdapter                 = DataAccess::IAdapter;
+using IChannelRepository       = DataAccess::IChannelsRepository;
+using IDirectMessageRepository = DataAccess::IDirectMessageRepository;
+using ILoginRepository         = DataAccess::ILoginRepository;
+using IMessagesRepository      = DataAccess::IMessagesRepository;
+using IRegisterRepository      = DataAccess::IRegisterRepository;
+using IRepliesRepository       = DataAccess::IRepliesRepository;
+using Query                    = MockQuery::MockQuery;
+
+struct TestAbstractRepository
+{
+public:
+    std::unique_ptr<Query>& getTable() { return _pTable; }
```

Comment:
For me it was quite confusing to see there "Query", not MockQuery. I know that's shorter than previous, but it still confusing